src/widget/mod.rs

```diff
@@ -0,0 +1,15 @@
+pub mod container;
+pub mod text;
+
+use embedded_graphics::prelude::*;
+
+pub trait Widget<Color> {
+    fn draw<Display, Error>(
+        &self,
+        display: &mut Display,
+        origin: Point,
+        size: Size,
```

Comment:
Are `Point` and `Size` passed not as references because they're `Copy`? Or does it make sense that `draw()` would take ownership of them?

Reply:
Yeah, they are both `Copy`, so I just let that happen. (I default to never passing `Copy` things by reference.)